### ISSUE TITLE
[Base-Connection Throttle] Connection Throttle ignores setting

### ISSUE THREAD

**Issue by lihad** [keepcalm/BukkitForge#316]
The "connection-throttle' setting in bukkit.yml ignores any value, uses default.

**Comment by charlieTheBotDev** [comment]
Oooooh that might be why #239 is broken too, you have to set the connection throttle to -1 to make the proxy work correctly according to the bukkitdev page.

**Comment by keepcalm** [comment]
Yes, most of bukkit.yml is ignored ATM, simply because forge doesn't have hooks to do this.
